nextflow.config

```diff
@@ -14,15 +14,18 @@ params {
 
     // References
     genome                     = 'GRCh38'
+    genome_version             = 37
```

Comment:
I would rename this to something like gencode_annotation_version. To me genome version implies genome build version ie 37 or 38

Reply:
That's true, it would be more clear